packages/langium/src/lsp/document-update-handler.ts

```diff
@@ -119,4 +142,25 @@ export class DefaultDocumentUpdateHandler implements DocumentUpdateHandler {
         this.fireDocumentUpdate(changedUris, deletedUris);
     }
 
+    didOpenDocument(_event: TextDocumentChangeEvent<TextDocument>): void {
```

Comment:
The interface methods are optional, so there's no need to provide empty implementations here.

Reply:
Rremoved.

Reply:
Removed no-op implementations.